### ISSUE TITLE
Missclassification of French BDPM items

### ISSUE THREAD

**Issue by cecileroseau** [OHDSI/Vocabulary-v5.0#681]
Hello, with my team (Health Data Hub), we found that many BDPM (Base de Données Publique des Médicaments) were missclassified as devices, whereas they are drugs according to [OHDSI definition](https://www.ohdsi.org/web/wiki/doku.php?id=documentation:vocabulary:drug). 

With my colleagues, we reviewed all items from BDPM which were classified as Device, to assert if the classification is right or if the item should be considered as Drug. Attached you'll find our classification of these codes : we added a column true_domain_id to the CONCEPT table and filled it when necessary. 

Doubt remain for two types  of item : 
- medical oxygen
- nitrous oxide

In France they are considered as Drugs, but we couldn't deduce from OHDSI's definition how to classify them to match OMOP-CDM standards. 

If you need any additionnal information, we're more than willing to help ! 

Cécile Roseau

[BDPM_device.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/9600452/BDPM_device.xlsx)

**Comment by cgreich** [comment]
@cecileroseau:

Thanks for this nice review. Always helps if the community checks, instead of blindly relying on the work of the vocab team.

However, many of what you listed in your spreadsheet are no drugs by the OHDSI definition. They are contrast agents, antiseptic perfusion liquids or radiopharmaceuticals. None of these are drugs, even though they look like it and have formulations like "solution for infusion". They are devices. But let us go through in detail.

How? Want to sit down?

**Comment by cecileroseau** [comment]
@cgreich True, many of this list are actual devices, and we left the column true_domain_id empty in this case. But lots of them are also surprinsingly missclassified and should be considered as drugs. For instance :
 ASPEGIC (10 different codes) is actually aspirin . I reviewed this list with our DIM doctor (a doctor in charge of coding hospital data in France), using OHDSI's definition of Drugs. 

But sure, we can go through in details with my colleague !

**Comment by mik-ohdsi** [comment]
Hi @cecileroseau  -  I have been going over the first 500 entries and added my assessment.
[BDPM_device_reviewed.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/9665289/BDPM_device_reviewed.xlsx)
It is true that there are some valid drug entries that are wrongly in the device domain. There are also a lot of entries now labeled as drugs that clearly are not. Could you go over the file and review my color coded assessment / assess the remaining ones yourself and boil down the list to items that, according to what you can see now as assessment, are really drugs?

**Comment by cecileroseau** [comment]
Hi @mik-ohdsi  ! 
Thank you for the review. We double-checked with my colleagues, and added three columns to the file : 
- HDH_double_check : 
green = we're sure it should be considered as a drug, 
red = we agree it is a device according to OHDSI's definition, 
yellow = it doesn't match either drug or device definition (mostly medicinal herbal teas are in that situation...) 
- Comment : arguments to show why the entry should be considered as a drug (for instance : we found an equivalent in RxNorm, it has an well known active ingredient, we can provide the indication for use of the drug, and others that make us think it is a drug -or not) 
- ATC class : we added ATC class to which the drug belongs when available. 
[BDPM_device_ohdsi_reviewed.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/9757404/BDPM_device_ohdsi_reviewed.xlsx)

Doubt remains for medicinal oxygen and nitrous oxide (and more generally for gases) , for which we really don't know if it should be considered as drug or device. 

Thank you for spending so much time on this issue !
